android/app/build.gradle

```diff
@@ -77,4 +84,5 @@ flutter {
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'androidx.multidex:multidex:2.0.1'
+    implementation 'com.github.yalantis:ucrop:2.2.6'
```

Comment:
Remove this and then check if cropper is working or not in release mode.

Reply:
It is working properly in release mode without this dependency.